pkg/webhook/policy.go

```diff
@@ -52,8 +35,26 @@ var (
 // below: notations add permissions kube-mgmt needs. Access cannot yet be restricted on a namespace-level granularity
 // +kubebuilder:rbac:groups=*,resources=*,verbs=get;list;watch
 // +kubebuilder:rbac:groups=,resources=configmaps,verbs=get;list;watch;create;update;patch;delete
-func AddPolicyWebhook(mgr manager.Manager) error {
-	opa := opa.NewFromFlags()
+func AddPolicyWebhook(mgr manager.Manager, opa opa.Client) error {
+	validatingWh, err := builder.NewWebhookBuilder().
+		Validating().
+		Name("validation.gatekeeper.sh").
```

Comment:
Let's use the `webhookName` flag instead of hardcoding it here. https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/policy.go#L48 we can default it to `validation.gatekeeper.sh` as you have here.

Reply:
Yeah, sorry for clobbering your change twice :/

Reply:
re-integrated